public/pages/AnomalyCharts/utils/constants.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+export const INITIAL_ANOMALY_SUMMARY = {
+    anomalyOccurrence: 0,
+    minAnomalyGrade: 0.0,
+    maxAnomalyGrade: 0.0,
```

Comment:
small - maybe change to `minGrade`/`maxGrade` to be consistent with `minConfidence`/`maxConfidence`